api.swagger.yaml

```diff
@@ -9,6 +9,23 @@ security:
   - APISessionHeader: []
   - APIKeyHeader: []
 paths:
+  /status:
+    get:
+      tags:
+        - status
+      responses:
+        '200':
+          description: Status returned successfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    description: Status of the API
+                    type: string
+                    example: 'Ok'
```

Comment:
If we are returning a string we should either document the possible values and what they mean, or if it's an unpredictable message intended to be read by humans, at least document that fact explicitly.